openmc/summary.py

```diff
@@ -64,6 +71,26 @@ def nuclides(self):
         return self._nuclides
 
     @property
+    def fast_materials(self):
```

Comment:
I'm suggesting that these not be publicly accessible for the time being. What I don't want to do is introduce new attribute/methods that we end up changing in the next release anyway, so by just having and using private `_fast_*` attributes, we can at least be sure that MGXS library loading is fast.